testapps/test_pillowtop/tests/test_settings.py

```diff
@@ -1,12 +1,13 @@
 import os
 from django.conf import settings
-from django.test import TestCase
+from django.test import TestCase, override_settings
 import json
 from corehq.util.test_utils import TestFileMixin
 from pillowtop.listener import AliasedElasticPillow, BasicPillow
 from pillowtop.utils import get_all_pillow_configs
 
 
+@override_settings(DEBUG=True)
```

Comment:
Seems fine for just this case but if we start using it like this more then
we should switch to the other approach.

On 11 November 2015 at 09:50, Cory Zue <notifications@github.com> wrote:

> In testapps/test_pillowtop/tests/test_settings.py
> <https://github.com/dimagi/commcare-hq/pull/9046#discussion_r44508545>:
>
> >  import json
> >  from corehq.util.test_utils import TestFileMixin
> >  from pillowtop.listener import AliasedElasticPillow, BasicPillow
> >  from pillowtop.utils import get_all_pillow_configs
> >
> >
> > +@override_settings(DEBUG=True)
>
> yeah, i actually added it at first yesterday and decided this was simpler.
> happy to add it back if you think that's better
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9046/files#r44508545>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa